src/main/java/org/mythicmc/editstick/listener/PlayerInteractListener.java

```diff
@@ -25,27 +24,7 @@ public void onPlayerInteract(PlayerInteractEvent e) {
                         p.hasPermission("editstick.use")) {
                     Location location = Location.at(p.getWorld().getName(), block.getX(),
                             block.getY(), block.getZ());
-                    if (location.isUnownedPlotArea()) {
-                        if (location.isPlotRoad() && (p.hasPermission("plots.admin.build.road")) ||
-                                p.hasPermission("plots.admin") ||
-                                p.hasPermission("plots.admin.build")) {
-                            BlockUtils.rotateBlock(block);
-                            e.setCancelled(true);
-                        } else if (!location.isPlotRoad() && (p.hasPermission("plots.admin.build.unowned") ||
-                                p.hasPermission("plots.admin") ||
-                                p.hasPermission("plots.admin.build"))) {
-                            BlockUtils.rotateBlock(block);
-                            e.setCancelled(true);
-                        }
-                        return;
-                    }
-                    Plot plot = location.getPlot();
-                    if (plot != null && (plot.getMembers().contains(p.getUniqueId()) ||
-                            plot.getTrusted().contains(p.getUniqueId()) ||
-                            plot.getOwners().contains(p.getUniqueId()) ||
-                            p.hasPermission("plots.admin.build.other") ||
-                            p.hasPermission("plots.admin") ||
-                            p.hasPermission("plots.admin.build"))) {
+                    if(EditStickUtils.canBuild(p, location)){
```

Comment:
Code style doesn't fit.
```suggestion
                    if (EditStickUtils.canBuild(p, location)) {
```